### ISSUE TITLE
'az aro create' --tags are not applied to aro-xxxxxxxx resource group

### ISSUE THREAD

**Issue by AWSmith0216** [Azure/OpenShift#232]
I'm creating an Azure Red Hat OpenShift cluster using 'az aro create'.  The creation works fine, and I get the 'OpenShift cluster' object in my resource group. On that object the '--tags' I pass in are set properly.

However, the 'aro-xxxxxx' resource group that is created to contain all the cluster resources does not have those tags set on the resource group or any of the objects it contains. My assumption would be that the provided tags would be applied to all of those generated objects.

**Comment by miroag** [comment]
:+1:

**Comment by dhatrifork** [comment]
This is a huge problem. Any time line for fix?

**Comment by bvenkataraman** [comment]
Created in 2021 and still not resolved, how does Redhat Team expect for enterprise customers to use this product with a fully policy management enabled on our subscription?

**Comment by jboutaud** [comment]
We just released documentation around how to get resources tagged through Azure policy. The documentation is located here:
https://learn.microsoft.com/en-us/azure/openshift/howto-tag-resources

**Comment by narendrakumar-nj** [comment]
The documentation shared in the previous comment is helpful only for adding tags post-creation. There might be policies in place that would prevent the creation of new resource groups without certain mandatory tags configured. In such cases, we wouldn't be able to create the cluster.

**Comment by gigabytte** [comment]
Solution on this? Many orgs enforce tagging at the RG level which prevents deployment. To exempt an RG just over a tagging policy is ridiculous and make post tagging cumbersome for cost tracking .. etc.

@SumanthMarigowda can we get an update on this? In my opinion this should be a day one feature, policy exemptions for tags is inexcusable in an enterprise env especially for an enterprise product like ARO.

**Comment by jboutaud** [comment]
This article and the steps associated with has been tested in environment where a policy prevents the creation of resources without a tag: https://learn.microsoft.com/en-us/azure/openshift/howto-tag-resources. 

The important parts of the article is to create the policy and apply it to the resource group name that will be created by the installation. Use the "--cluster-resource-group" to specify the name which will be used to hold all the resources and matches the policy. The policy would mutate all the requests creating resources in that resource group to have the tags specified

If it doesn't work, please open a support case

**Comment by gigabytte** [comment]
@jboutaud Yes this would the work around going forward however for ease of use this should be done automatically by Azure. For example AKS applies all tags from the resource RG into the RG for the cluster resources.
